v22.1/add-super-region.md

```diff
@@ -0,0 +1,115 @@
+---
+title: ADD SUPER REGION
+summary: The ADD SUPER REGION statement adds a super region made up of a set of regions such that data from regional tables will be stored in those regions. 
```

Comment:
I'd explicitly call out regional by row tables as well. 
Also I'd say such that if the regional table's region or regional by row partition region is part of the super region, then all data from that table / partition will be stored in that super region.

Reply:
Added mention of RBR

I modified your suggested text slightly to say regional by row's 'home region' since even tho they use partitions the normie RBR user is supposed to not have to care about partitions too much (which is what we tell them at least on https://www.cockroachlabs.com/docs/dev/partitioning.html)